client/src/main/java/io/pravega/client/segment/impl/SegmentOutputStreamImpl.java

```diff
@@ -509,13 +509,13 @@ public void flush() throws SegmentSealedException {
             }
             state.waitForInflight();
             Exceptions.checkNotClosed(state.isClosed(), this);
-            /* SegmentSealedException is thrown if either of the below conditions are true
+        }
```

Comment:
I'm not sure I understand the need for this change. Is there any case this prevents a problem?

Reply:
Yes this change is not required since we remove the segment writers only after the inflight events are resent to the new segments.

Reply:
Yes, it is not required.